views/partials/index/about.ejs

```diff
@@ -1,13 +1,20 @@
 <div class="section about-section">
-  <div id="about" class="container" style="height:100%">
+  <div id="about" class="container">
     <div class="about-content row justify-content-between">
-      <div class="col-lg-6 col-md-6 col-sm-12 d-flex flex-column justify-content-center">
-        <h1 class="section-title about-title pb-3">
-          About Us
-          </h2>
-          <p class="about-description">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla dapibus id est nec molestie. Phasellus sit amet finibus enim. Sed aliquet mauris a velit ultricies, et ultricies velit malesuada. Morbi quis sodales velit. Curabitur tristique mauris sapien, sed consequat diam sollicitudin ac. Aliquam erat volutpat. Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-          </p>
+      <div class="col-lg-6 col-md-6 col-sm-12 d-flex flex-column">
+        <div class="container px-0">
+          <h1 class="section-title about-title">
+            About
+          </h1>
+        </div>
+        <br>
+        <h5>
+          TAMU Datathon is the World's First & Only MLH Data Science Hackathon!
+        </h5>
+        <br>
```

Comment:
I think this should be reverted to a `<p>` and we should remove `<br>` as the `<h5>` uses Roboto Mono which doesn't look right as a description. We can add a `font-weight: 500` style to the p tag to keep it bold, however.

![image](https://user-images.githubusercontent.com/7832610/80066231-2f1b4800-8501-11ea-83dc-fad64e715077.png)
```suggestion
        <p class="about-description" style="font-weight: 500">
          TAMU Datathon is the World's First & Only MLH Data Science Hackathon!
        </p>
```